webpack/scenes/AlternateContentSources/Create/Steps/AcsUrlPaths.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  TextArea,
+  Switch,
+} from '@patternfly/react-core';
+import ACSCreateContext from '../ACSCreateContext';
+import WizardHeader from '../../../ContentViews/components/WizardHeader';
+
+const AcsUrlPaths = () => {
+  const {
+    url, setUrl, subpaths, setSubpaths, verifySSL, setVerifySSL,
+  } = useContext(ACSCreateContext);
+
+  return (
+    <>
+      <WizardHeader
+        title={__('URL and paths')}
+        description={__('Enter in the base path and any subpaths that should be searched for alternate content.' +
+          'The base path can be an HTTP network location or a filesystem location.')}
```

Comment:
We need a space before "The base path..."

Also, maybe we could just say "...can be a web address or a filesystem location...".  I feel like it's a little inconsistent seeing HTTP and then https:// below in the Base URL box.

Reply:
Updated with space and changed wording.